cargo-workspaces/src/utils/pkg.rs

```diff
@@ -46,19 +46,17 @@ impl Listable for Vec<Pkg> {
             let mut width = first - pkg.name.len();
 
             if list.long {
-                let path = if pkg.path.is_empty() { "." } else { &pkg.path };
```

Comment:
Removing this line has failed the tests.

Reply:
I re-added the check to replace the empty path with `.`.